department-of-veterans-affairs/abd-vro
# PDF Generator: Consumer

# PDF Generator: Consumer 

## Description

**NOTE:** This is part of the work from #108 which is being broken down into more manageable PRs. 



### What was the problem?

There was a Ruby PDF Generator that needed to be replaced with a Python version

### How does this fix it?

This first PR sets up the consumer for the PDF generator. The only functionality at the moment is connecting to the queue and receiving and logging any messages

## How to test this PR

- `./gradlew clean`
- `./gradlew build check docker`
- `./gradlew :app:dockerComposeDown :app:dcPrune :app:dockerComposeUp`
- Once all containers are up, navigate to service-python/examples
- Run `python generate_pdf.py`
- View docker logs for see that the consumer received the message with the same data that `generate_pdf.py` is passing.
